.github/workflows/scripts/e2e/gmc_install.sh

```diff
@@ -36,6 +36,7 @@ function copy_manifests() {
     # Copy manifest into gmc
     mkdir -p $(pwd)/config/manifests
     cp $(dirname $(pwd))/manifests/ChatQnA/*.yaml -p $(pwd)/config/manifests/
```

Comment:
remove coping from ChatQnA after all manifests finalized

Reply:
yes, that is planned